src/web/core/src/test/java/org/geoserver/web/GeoServerWicketTestSupport.java

```diff
@@ -118,7 +123,7 @@ public Component findComponentByContent(MarkupContainer root, Object content, Cl
         root.visitChildren(componentClass, finder);
         return finder.candidate;
     }
-    
+
```

Comment:
This file seems to contain only non-functional changes

Reply:
Yep, will remove it from changeset